docs/reference/module-options.md

```diff
@@ -372,3 +372,457 @@ module.exports = {
+| [`builders`](#builders) | Object | Set query builder values to be used when pages are served. |
+| [`home`](#home) | Boolean/Object | Change how the home page is added to `req.data` when pages are served. |
+| [`minimumPark`](#minimumpark) | Array | Override default parked pages, including the home page. |
+| [`park`](#park) | Array | Set pages to be created on site start with configuration. |
+| [`publicApiProjection`](#publicapiprojection-for-pages) | Object | Set query builder values to be used when pages are served. |
+| [`quickCreate`](#quickcreate-for-pages) | Boolean | Set to `false` to remove pages from the quick create menu. |
+| [`types`](#types) | Array | Set the page types available for new pages. |
+
+### `builders`
+
+<!-- TODO: Update link to a more detailed explanation of builders when
+available. -->
+The `builders` option can be used to apply any existing [query builders](/guide/major-changes.md#queries) when a page is served by its URL. This effects the data available on the page object, `req.data.page` (`data.page` in templates).
+
+The default value is:
+```javascript
+{
+  children: true,
+  ancestors: { children: true }
+}
+```
+
+In this example, page objects are loaded with one level of page tree "children" as `_children` and their "ancestor" pages, each with one level of their child pages, on `_ancestors`.
+
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    builders: {
+      children: { depth: 2 }
+    }
+  },
+  // ...
+}
+```
+
+In this example, we are not including ancestor pages and are requesting two levels of child pages (direct children and their direct children).
+
+### `home`
+
+The home page document is added to all page requests on `req.data.home` so it can be referenced in all page templates. That home page object also includes a `_children` property containing an array of top level page objects. `home` option settings to offer minor performance improvements for large sites include:
+
+| Setting | Description |
+|---------|-------------|
+| `false` | Disables adding the home page document to the requests. |
+| `{ children: false }` | Includes the home page document, but without the child pages array. If the [`builders` option](#builders) has an `ancestors` property, that will take precedence. |
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/page/index.js
+module.exports = {
+  options: {
+    home: { children: false }
+  },
+  // ...
+}
+```
+
+### `minimumPark`
+
+The `minimumPark` option offers the ability to overwrite how the default required pages are configured. This includes the home page and the trash "page" (trash is part of the page tree). Most "parking" pages should be done with the [`park`](#park) option.
+
+::: warning
+Use with caution. Removing either of those pages as required with this option would have negative effects, but this does offer the ability to change the title or page type of the home page, for example.
```

Comment:
"Use with caution. Removing either of those pages as required with this option would have negative effects" This statement is a bit confusing. "Would have negative effects". We're also leading with the warning. It may make sense to lead with the function, and follow with the warning.